app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self, name: str, health: int = 100, hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
```

Comment:
Better create a class method and use it there, like self.add_animal (add_animal cls method)